chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -322,6 +322,9 @@ def find_node_boot_mac_addresses(node, admin_data_net)
             cloud_available = true if name.include? "Cloud"
           end
 
+          ntp_servers = search(:node, "roles:ntp-server")
+          ntp_servers_ips = ntp_servers.map { |n| Chef::Recipe::Barclamp::Inventory.get_network_by_type(n, "admin").address }
```

Comment:
Just a hint: you don't need the `Chef::Recipe::` prefix (but that won't help with the hound warning)